ironfish-cli/src/commands/chain/repair.ts

```diff
@@ -153,14 +153,16 @@ export default class RepairChain extends IronfishCommand {
     let block = header ? await node.chain.getBlock(header) : null
     let prev = await node.chain.getHeaderAtSequence(TREE_START - 1)
     const noteSize = prev && prev.noteSize !== null ? prev.noteSize : 0
-    const nullifierSize = prev && prev.nullifierSize !== null ? prev.nullifierSize : 0
 
     CliUx.ux.action.start('Clearing notes MerkleTree')
     await node.chain.notes.truncate(noteSize)
     CliUx.ux.action.stop()
 
-    CliUx.ux.action.start('Clearing nullifier MerkleTree')
-    await node.chain.nullifiers.truncate(nullifierSize)
+    CliUx.ux.action.start('Clearing nullifier set')
+
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
```

Comment:
Don't add this.

Reply:
Fair, what do you think we should do? I don't really want to write a clear method on nullifier set because it's not used anywhere else and this command might be out of date eventually. We could delete this command also or not repair the nullifier set in it.

Reply:
Added a `clear` method here since we are keeping this repair command